media/live-seekable-range.html

```diff
@@ -0,0 +1,37 @@
+---
+feature_name: MediaSource.setLiveSeekableRange
+chrome_version: 62
+feature_id: 5671401352593408
+check_min_version: true
+---
+
+<h3>Background</h3>
+<p>
+This API lets MediaSource apps more effectively customize the
+<code>HTMLMediaElement.seekable</code> range logic by providing (or removing) a
+single seekable range that is union'ed with the current buffered ranges to
+result in a single seekable range which fits both, when media duration is
+<code>+Infinite</code>.
+</p>
+
+<button id="setButton">mediaSource.setLiveSeekableRange(40, 42)</button>
+<button id="clearButton">mediaSource.clearLiveSeekableRange()</button>
+<button id="removeButton">sourceBuffer.remove(3, 6)</button>
+
+{% include output_helper.html initial_output_content=initial_output_content %}
+
+{% include js_snippet.html filename='live-seekable-range.js' %}
+
+<script>
+  document.querySelector('#setButton').addEventListener('click', function() {
+    onSetLiveSeekableRangeButtonClick();
+  });
+  document.querySelector('#clearButton').addEventListener('click', function() {
+    onClearLiveSeekableRangeButtonClick();
+  });
+  document.querySelector('#removeButton').addEventListener('click', function() {
+    onRemoveMediaSegmentButtonClick();
+  });
+
```

Comment:
Three more buttons please:
1) setLiveSeekableRange(1,4)  --> result, if there's still buffered [3,6) and +Infinity duration, should be seekable [1,6)  --> This shows the front part of the "union" operation is working. (And if nothing buffered, should be just [1,6) if +Infinity duration).
2) setInfiniteDuration() --> mediaSource.duration = +Infinity; and then log result..
3) setFiniteDuration() --> mediaSource.duration = some finite value >= highest buffered end time. Maybe 6?

We might also want to add endOfStream() and fetchAndAppend3To6Media() to show the interesting behavior resulting from those calls.